app/watch/page.tsx

```diff
@@ -0,0 +1,483 @@
+	let url = `https://youtube.com/watch?v=${id}`
+
+	if (startTime && startTime !== "0s") {
+		url += `&t=${startTime}`
+	}
+
+	return url
+}
+
+const generateHeroDescription = (talks: Talk[]) => {
+	return (
+		<>
+			San Diego DevX Monthly Meetup live talks featuring:
+			<br />
+			<br />
+			{talks.map((talk, index) => (
+				<span key={index}>
+					• <SpeakerName>{talk.speaker}</SpeakerName> - <SpeakerName>{talk.title}</SpeakerName>
+					<br />
+				</span>
+			))}
+			<br />
+			Join our discord and see events at https://devx.network/
+		</>
+	)
+}
+
+// Components
+
+export default function Watch() {
+	// Group livestreams by year for sectioned display
+	const videosByYear = livestreams.reduce(
+		(acc: Record<number, Livestream[]>, video) => {
+			if (!acc[video.year]) acc[video.year] = []
+			acc[video.year].push(video)
+			return acc
+		},
+		{} as Record<number, Livestream[]>
+	)
+
+	// Sort years in descending order (most recent first)
+	const years = Object.keys(videosByYear)
+		.map(Number)
+		.sort((a, b) => b - a)
+
+	// Get the most recent video across all years
+	const allVideos = [...livestreams].sort(
+		(a, b) => new Date(b.date).getTime() - new Date(a.date).getTime()
+	)
+	const mostRecentVideo = allVideos[0]
+
+	return (
+		<>
+			<BackgroundContainer>
+				<ErrorBoundary
+					fallback={<div style={{ backgroundColor: "black", width: "100%", height: "100%" }} />}
+				>
+					<PotionBackground />
+				</ErrorBoundary>
+			</BackgroundContainer>
+			<Main>
+				{/* Hero Section with Most Recent Video above the fold */}
+				<HeroSection>
+					<HeroContent>
+						<HeroHook>Check out our most recent talk:</HeroHook>
+					</HeroContent>
+					<HeroVideoCard>
+						<HeroThumbnailSection>
+							<HeroThumbnailContainer>
+								<HeroVideoCategory>{mostRecentVideo.title}</HeroVideoCategory>
+								<ThumbnailLink
+									href={buildYouTubeUrl(mostRecentVideo.id, mostRecentVideo.startTime)}
+									target="_blank"
+									rel="noopener noreferrer"
+								>
+									<HeroThumbnailImage src="/images/DEVxCover.png" alt="" />
+									<HeroPlayButton>▶</HeroPlayButton>
+								</ThumbnailLink>
+							</HeroThumbnailContainer>
+						</HeroThumbnailSection>
+						<HeroVideoContent>
+							<HeroVideoDescription>
+								{generateHeroDescription(mostRecentVideo.talks)}
+							</HeroVideoDescription>
+						</HeroVideoContent>
+					</HeroVideoCard>
+				</HeroSection>
+
+				<WatchSection>
+					{years.map((year) => {
+						// Filter out the most recent video across all years from each year's display
+						const yearVideos = videosByYear[year].filter((video) => video.id !== mostRecentVideo.id)
+
+						// Skip year section if no videos remain after filtering out most recent
+						if (yearVideos.length === 0) return null
+
+						return (
+							<YearSection key={year}>
+								<YearHeader>{year}</YearHeader>
+
+								{/* Render all remaining videos in grid */}
+								<LivestreamGrid>
```

Comment:
This stream grid should support individual video's with one speaker per video. Perhaps limit the video list to the ones that are individual talk videos for now and omit the live stream videos. I really want to show case our videos as individual talks, not live streams.